src/client/datascience/notebook/notebookEditor.ts

```diff
@@ -159,6 +159,30 @@ export class NotebookEditor implements INotebookEditor {
             ).then(noop, noop);
         }
     }
+    public toggleOutput(): void {
```

Comment:
Suggestion only: add a test for this behavior so we avoid regressions in future.